docs/kubernetes-deployment/signer/values/mainnet/values-signer0-mainnet.yaml

```diff
@@ -32,7 +35,7 @@ env:
     format: stackdriver
     level: trace
   odis:
-    odisSignerTimeout: "6000"
+    odisSignerTimeout: "10000"
     shouldMockAccountService: "false"
     shouldMockRequestService: "false"
     testQuotaBypassPercentage: "50"
```

Comment:
should be 0

Reply:
Sorry, this comment was meant for `testQuotaPercentage:"50"`. 

`testQuotaPercentage:"50"` can actually be removed from the chart.